src/chain/chain.rs

```diff
@@ -753,7 +730,7 @@ impl Chain {
     fn should_skip_accumulator(&self, event: &NetworkEvent) -> bool {
         // FIXME: may also need to handle non SI votes to not get handled multiple times
         let si = match event.payload {
-            AccumulatingEvent::SectionInfo(ref si) => si,
+            AccumulatingEvent::SectionInfo(ref si) | AccumulatingEvent::NeighbourInfo(ref si) => si,
```

Comment:
It seems the `AccumulatingEvent::NeighbourInfo` is always being handled as the same to `AccumulatingEvent::SectionInfo`.
Then why have this extra enum ?

Reply:
This was done as the first step to make them different. But as I was modifying the code I found fixing the route cause did not require it, so I did not progress it further. However, multiple place would be clearer if handled differently as mentioned in PR description. As well, it helps with logs.

Reply:
ok.
I'd still prefer the `NeighbourInfo` to be in a separate PR.
However won't make this as a blocking point.